projects/dotcms-field-elements/src/components/dot-binary-file/dot-binary-file.tsx

```diff
@@ -84,12 +84,21 @@ export class DotBinaryFileComponent {
     @Prop({ reflectToAttr: true })
     buttonLabel = 'Browse';
 
+    /** (optional) Name of the file uploaded */
+    @Prop({ reflectToAttr: true, mutable: true })
+    previewImageName = '';
+
+    /** (optional) URL of the file uploaded */
+    @Prop({ reflectToAttr: true, mutable: true })
+    previewImageUrl = '';
+
     @State() status: DotFieldStatus;
 
+
     @Event() valueChange: EventEmitter<DotFieldValueEvent>;
```

Comment:
```suggestion
    @Event() valueChange: EventEmitter<DotFieldValueEvent<File | string>>;
```

Reply:
Also what happen with `DotBinaryFileEvent`? I think we have some typing issues here.